nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/AbstractCommentManager.java

```diff
@@ -166,4 +167,20 @@ protected void setCommentPermissions(CoreSession session, DocumentModel document
         return ancestorIds;
     }
 
+    /**
+     * Notifies the event of type {@code eventType} on the given {@code commentDocumentModel}.
+     *
+     * @param session the session
+     * @param commentDocumentModel the document model of the comment
+     * @param eventType the event type to fire
+     * @implSpec This method uses internally {@link #notifyEvent(CoreSession, String, DocumentModel, DocumentModel)}
+     * @since 11.1
+     */
+    protected void notifyEvent(CoreSession session, DocumentModel commentDocumentModel, String eventType) {
```

Comment:
i didn't follow the same signature because i think it more logical to have a kind of order when we have to pass parameters here `CoreSession -> DocumentModel -> eventType`

Reply:
I don't agree with you on logic, what we want is firing an event, and eventType is the only mandatory information (as we can have event without document model). Furthermore, this is not the signature we already have in this class, in `AbstractSession`, in `AbstractTrashService`, in `CommentsModerationServiceImpl`... In the end I prefer to be consistent with previous design + having event id in front of `DocumentModel` as it is more mandatory than model.